Common/Statistics/Statistics.cs

```diff
@@ -517,7 +517,13 @@ public static double Alpha(List<double> algoPerformance, List<double> benchmarkP
         /// <returns>Value for tracking error</returns>
         public static double TrackingError(List<double> algoPerformance, List<double> benchmarkPerformance)
         {
-            return Math.Sqrt(AnnualVariance(algoPerformance) - 2 * Correlation.Pearson(algoPerformance, benchmarkPerformance) * AnnualStandardDeviation(algoPerformance) * AnnualStandardDeviation(benchmarkPerformance) + AnnualVariance(benchmarkPerformance));
+            var performanceDifference = new List<double>();
```

Comment:
If arrays have different length will this explode? Try to always input check and not assume inputs are valid

Reply:
It would. Elsewhere there seems to be an assumption that array lengths are the same, but I'll add in some error handling. Would we want to return a specific value if they aren't the same length?

Reply:
You can use your best judgment here.

Reply:
I'm going to have it return NaN. Pretty much all of the statistics rely on the assumption that the algo and benchmark lists are of equal length, so un-equal lists would blow up a number of statistics. We could add in error handling or I could dig deeper into how these are used in Lean to see if we handle these possible cases elsewhere.